contracts/colony/Colony.sol

```diff
@@ -37,6 +37,15 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     return token;
   }
 
+  function makeArbitraryTransaction(address _to, uint256 _value, bytes memory _action)
+  public stoppable auth
+  returns (bool)
+  {
+    require(_to != colonyNetworkAddress, "colony-cannot-target-network");
+    require(_to != tokenLockingAddress, "colony-cannot-target-token-locking");
```

Comment:
It can't be a call to ERC20 `transfer()` otherwise internal bookkeeping will be messed up (strictly, it could be, if it hasn't been claimed yet, but seems all to easy to grief).

Do we need to prevent it making calls to extensions?

Reply:
Great point, that's a good restriction. Extensions don't have any `calledByColony` modifiers, so I don't think that's something we need to worry about.